### DISCUSSION TITLE
Target [Illuminate\Database\Eloquent\Model] is not instantiable

### DISCUSSION THREAD

**Discussion by bhattji** [filamentphp/filament#13798] [Help]
### Package

Form builder

### Package Version

3.2.x

### How can we help you?

I have the UserResource, which I guess, working all fine sometime before, however I find now I am not abe to create a new User (Target [Illuminate\Database\Eloquent\Model] is not instantiable.)

The EditUser works fine though

How to solve this?

Please help asap

**Comment by rvzug** [comment]
You are probably writing ```new Model``` somewhere, or passing the ```Model::class``` somewhere, where it later gets instantiated. It should be some model-class that extends the Model. This error doesn't look like a Filament issue, but a basic PHP understanding.
